docs/userguide/networking/dockernetworks.md

```diff
@@ -293,7 +293,9 @@ specifications.
 You can create multiple networks. You can add containers to more than one
 network. Containers can only communicate within networks but not across
 networks. A container attached to two networks can communicate with member
-containers in either network.
+containers in either network. When a container is connected to multiple
+networks, its external connectivity is provided via the first in
```

Comment:
Thanks.
Rephrased as adviced, but preferred to be consistent with golang docs, so used "lexical".
Let me know.